resources/js/visual-editor/editor/editor-app.css

```
@@ -217,3 +217,70 @@
     line-height: 1.5;
     color: var(--color-base-content, #374151);
 }
+
+/* #621 — composed-view chrome preview panels. Inert outline around the
+   block canvas; not selectable, no block-editor state. */
+.ap-visual-editor__chrome-preview {
+    margin: 12px 24px;
+    padding: 12px 16px;
+    border: 1px dashed color-mix(in oklab, currentColor 25%, transparent);
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)